
class TryCatchFinallyClass {
    fun main(){

        try {
            //throw Exception()
       // }catch (e : Exception){
          //  println(e.toString())
        }finally {

        }
    }
}